docs/dbpedia-spotlight-wrapper/v1.0/metadata.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "Dbpedia Spotlight Wrapper",
+  "description": "Apply named entity linking to all text documents in a MMIF file.",
+  "app_version": "v1.0",
+  "mmif_version": "1.0.0",
+  "analyzer_version": "version_1.0",
+  "app_license": "Apache 2.0",
+  "analyzer_license": "Apache 2.0",
+  "identifier": "http://apps.clams.ai/dbpedia-spotlight-wrapper/v1.0",
+  "url": "https://github.com/clamsproject/app-dbpedia-spotlight-wrapper",
+  "input": [
+    {
+      "@type": "http://mmif.clams.ai/vocabulary/TextDocument/v1",
+      "required": true
+    }
+  ],
+  "output": [
+    {
+      "@type": "http://vocab.lappsgrid.org/NamedEntity"
+    }
+  ],
+  "parameters": [
+    {
+      "name": "confidence",
+      "description": "disambiguation confidence score for linking",
+      "type": "number",
+      "default": 0.5,
```

Comment:
found the app is using SDK 1.0.3, and having a `confidence` parameter default value as a real number. This will be a problem with this bug: https://github.com/clamsproject/clams-python/issues/173 . To mitigate this, either update the SDK version to 1.0.7 (I don't think moving from 1.0.3 to 1.0.7 would break anything) or change the default value from `0.5` to `"0.5"`.